nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Create new Category

# Description
Added the ability to create a new category. Edit and Delete on categories are still non-functional.
Fixes #10 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. Run SQL scripts 1 and 2 in the SQL folder to generate seed data
2. Run the program through the VS debugger or in CLI ```dotnet run```
3. Once the browser is available, login using ```admin@example.com```
4. Navigate to Admin Tools > Category Management
5. Click the 'Create Category' button at the top of the category list
6. Type in the name of a new category. The field should restrict the number of characters to 50.
7. Save the new category. This will take you to the category list, where the new category should be displayed alphabetically.
# Checklist:
